frontend/src/app/modules/album/components/create-edit-album/edit-album-modal/edit-album-modal.component.ts

```diff
@@ -58,15 +61,37 @@ export class EditAlbumModalComponent implements OnInit {
       .subscribe({
         next: (data) => {
           this.groups = data;
+
+          if (this.groups?.length) {
+            this.fillAuthors(this.groups);
+          }
         }
       });
   };
 
+  fillAuthors = (groups: Group[]) => {
+    const element = this.selectElement.nativeElement;
+
+    $(element).append($('<option></option>').val(0).text(this.userName));
+
+    groups.forEach((group) => {
+      $(element).append($('<option></option>').val(group.id).text(group?.name));
+    });
+
+    $(element).val(this.editedAlbum.groupId ?? 0);
+
+    this.isAuthorHidden = false;
+  };
```

Comment:
Is it possible to get rid of DOM js code?
Possibly you should form an array with available values firstly and then show them in select via ngFor. Try to change it.